src/CommandAndStateMessageParser.cpp

```diff
@@ -484,9 +484,12 @@ samples::RigidBodyState CommandAndStateMessageParser::getRevolutionPoseZAttitude
     samples::RigidBodyState pose;
     pose.time = Time::now();
     pose.position.z() = -state_z;
+    pose.cov_position(2, 2) = 1e-1;
```

Comment:
what are these parameters in cov_position?

Reply:
cov_position is a matrix, so this is the element in the second row and column